content/en/docs/tasks/extend-kubernetes/socks5-proxy-access-api.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Use a SOCKS5 Proxy to Access the Kubernetes API
+content_type: task
+weight: 42
+min-kubernetes-server-version: v1.24
+---
+
+<!-- overview -->
+This page shows how to use a SOCKS5 proxy to access the API of a remote Kubernetes cluster.
+This is useful when the cluster you want to access does not expose its API directly on the public internet.
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+<!-- steps -->
+
+## Overview
+
+{{< note >}}
+This example tunnels traffic using SSH, with the SSH client and server acting as a SOCKS proxy.
+You can instead use any other kind of [SOCKS5](https://en.wikipedia.org/wiki/SOCKS#SOCKS5) proxies.
+{{</ note >}}
+
+Figure 1 represents what we're going to achieve in this tutorial.
```

Comment:
```suggestion
Figure 1 represents what you're going to achieve in this task.
```
If this is a tutorial, it should be in the tutorial section. I think this _is_ a task page.